node_modules/.package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "storybook-1",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
please remove these, we do not use npm-cli in this repo

Reply:
Thanks for your review! I removed this file now.

https://github.com/storybookjs/storybook/pull/20420/commits/25b128123c92c7d410234a4c1d6b02f20d4bf1ef